### PR TITLE
Cms infrastructure

### PR DESCRIPTION
**Pull request by zackaad**
We would like to push merge our code with the main branch so others can utilize the code.

### CODE REVIEW

File: src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/HTMLSite.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package dk.sdu.se_f22.contentmodule.infrastructure.domain;
+import dk.sdu.se_f22.
```
Comment (MagnusHJensen) [inline src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/HTMLSite.java:2]: Unclosed import statement.

**Comment by marc7636** [comment]
Can i ask you guys to update your branch to the latest commit and fix the conflicts with the main branch?

File: src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/HTMLSite.java (hunk 1 above)
Comment (MagnusHJensen) [reply src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/HTMLSite.java]: Checked out locally, can't see this import.

File: src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java [hunk 2]
```diff
@@ -35,9 +38,21 @@ public String getFilteredTokens() {
...
+        }
+        return tokenStrings;
+    }
+
     ArrayList<Token> filterTokens(ArrayList<Token> tokens) {
+        tokenToString(tokens);
         //tokens.set(classstopwords.methodprovided(tokens));
-        //tokens.set(classirr.methodprovided(tokens));
+        //tokens.set(classirr.searchForIrregularWords(tokens));
         //tokens.set(classstem.methodprovided(tokens));
```
Comment (MagnusHJensen) [inline src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java]: `classstem.methodprovided(tokens)` can be replaced with: `new Stemmer().stem(tokens)`

File: src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java [hunk 3]
```diff
@@ -27,32 +27,35 @@ public FilteredTokens(ArrayList<Token> tokens) {
...
 
+
+
     public FilteredTokens(int idToken, String filteredTokens){
         this.idToken = idToken;
         this.filteredTokens = filteredTokens;
     }
 
-    public int getIdToken() {
+    int getIdToken() {
```
Comment (MagnusHJensen) [inline src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java:37]: Meant to be default and not private modifier?

File: src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java [hunk 4]
```diff
@@ -27,32 +27,35 @@ public FilteredTokens(ArrayList<Token> tokens) {
...
         return idToken;
     }
 
-    public String getFilteredTokens() {
+    String getFilteredTokens() {
```
Comment (MagnusHJensen) [inline src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java:41]: Meant to be default and not private modifier?

File: src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java [hunk 5]
```diff
@@ -27,32 +27,35 @@ public FilteredTokens(ArrayList<Token> tokens) {
...
         return filteredTokens;
     }
 
-    private ArrayList<String> tokenToString(ArrayList<Token> tokens) {
+    static ArrayList<String> tokenToString(ArrayList<Token> tokens) {
```
Comment (MagnusHJensen) [inline src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java:45]: Meant to be default and not private modifier?

File: src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java (hunk 4 above)
Comment (zackaad) [reply src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java:41]: done

File: src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java (hunk 3 above)
Comment (zackaad) [reply src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java:37]: done

File: src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java (hunk 2 above)
Comment (zackaad) [reply src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java]: done

File: src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java (hunk 5 above)
Comment (zackaad) [reply src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java:45]: done

**Pull request merged**